Tasks/UseDotNetV2/globaljsonfetcher.ts

```diff
@@ -64,6 +65,16 @@ export class globalJsonFetcher {
 
 }
 
-class GlobalJson {
-    public sdk: { version: string };
+export class GlobalJson {    
+    constructor(version: string | null = null) {        
+        if(version != null){
+            this.sdk = new sdk();
+            this.sdk.version = version;
+        }
+    }
+    public sdk: sdk;
```

Comment:
Can be directly put version here ?

Reply:
no we can't. This is the structure of the `global.json` json inside a `global.json` file.